src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -246,7 +226,7 @@ exports[`Layer animation false 1`] = `
   outline: none;
   pointer-events: all;
   z-index: 15;
-  position: fixed;
+  position: absolute;
```

Comment:
I noticed a couple of potential backward compatibility issues on `position` value that replaces 'fixed' with 'absolute'.

Reply:
Correct, that's because we need the LayerContainer to be positioned with respect to the outer Layer and not with respect to the whole window.